packages/bitcore-client/bin/wallet-tx

```diff
@@ -23,7 +21,8 @@ const main = async () => {
   let wallet;
   try {
     wallet = await Wallet.loadWallet({ path });
-    let params = ({ addresses, amount, utxos, change } = program);
+    let params = ({ addresses, amount, utxos, change, fee } = program);
+    params.addresses = params.addresses.includes('[') ? JSON.parse(params.addresses) : params.addresses;
```

Comment:
This is to determine if it's a json string like '[{}]'
Better way?

Also, should be 
```
params.addresses = addresses.includes('[') ? JSON.parse(addresses) : addresses
```